clients/python/tracker_client/formatting.py

```diff
@@ -140,29 +141,73 @@ def format_all_results(result):
     # Extract the contents of the list of edges for guidance tags
     for x in result["findDomainByDomain"]["web"].keys():
```

Comment:
Can you make "x" more descriptive?

Reply:
For sure, its value is "https", "ssl", "dmarc" etc, so something like scan_category or result_category?

Reply:
yup, your suggestion works fine